manifests/opa.yaml

```diff
@@ -107,10 +107,13 @@ spec:
           args:
             - "run"
             - "--server"
+            - "--config-file=/config/config.yaml"
             - "--tls-cert-file=/certs/tls.crt"
             - "--tls-private-key-file=/certs/tls.key"
             - "--addr=0.0.0.0:443"
             - "--addr=http://127.0.0.1:8181"
+            - "--log-format=json-pretty"
+            - "--set=decision_logs.console=true"
```

Comment:
I think this will need to be plumbed through as well - i.e. `opa.decisionLogs: true`